bot/events.go

```diff
@@ -123,8 +124,19 @@ func botCommand(botCtx cmd.BotContext, slackCtx cmd.SlackContext) {
 	}
 }
 
-func checkLanguage(botClient *slack.Client, event *slackevents.MessageEvent) {
-	if reply := inclusion.Filter(event.Text); reply != "" {
-		_ = slackx.SendEphemeral(botClient, event.ThreadTimeStamp, event.Channel, event.User, reply)
+func checkLanguage(botCtx cmd.BotContext, event *slackevents.MessageEvent) {
+	if filter := inclusion.Filter(event.Text); filter != nil {
```

Comment:
nitpick: reverse condition and early return to avoid indenting

Reply:
fixed